torch_xla/csrc/tensor_methods.cpp

```diff
@@ -864,52 +863,39 @@ void XLATensor::bernoulli_(XLATensorPtr& input,
       input->shape().get()));
 }
 
-XLATensorPtr XLATensor::bitwise_and(const XLATensorPtr& input,
-                                    const at::Scalar& other) {
-  CheckIsIntegralOrPred(input->shape(), "__and__");
-  torch::lazy::Value constant =
-      GetIrValueForScalar(other, input->shape(), input->GetDevice());
-  return input->CreateFrom(BitwiseAnd(input->GetIrValue(), constant));
-}
-
 XLATensorPtr XLATensor::bitwise_and(const XLATensorPtr& input,
```

Comment:
we should remove `XLATensor::bitwise_xxx` since they are not needed anymore

Reply:
Done